### ISSUE TITLE
Add alt text to images

### ISSUE THREAD

**Issue by irisvw** [irisvw/user-experience-enhanced-website#10]
## Description
All images should have alt text for accessibility reasons.

## Solution
In the Liquid template card, set the alt text to the ``image.title`` property.
https://fdnd-agency.directus.app/items/tm_story?fields=image.title,image.id

## To do
- [x] Alt text for story images
- [x] Alt text for playlist images

**Comment by irisvw** [comment]
Fixed in c093ef16e4ad6fb8de0b539f377b63f4c47ec73d

Playlist images:
https://github.com/irisvw/user-experience-enhanced-website/commit/c093ef16e4ad6fb8de0b539f377b63f4c47ec73d#diff-56c91bf6bae24ff23a9add6b2dde2d46f828eeb4119b09f855eff6e3c117a041

Story images:
https://github.com/irisvw/user-experience-enhanced-website/commit/c093ef16e4ad6fb8de0b539f377b63f4c47ec73d#diff-078a91a74066d87654420788278f8bf804ad6ba21a616f1d0788ec46fa8a4294
